src/integrations/rewriteframes.ts

```diff
@@ -8,40 +11,50 @@ import { getCurrentServerUrl } from '../utils/webViewUrl';
  * which appends app:// to the beginning of the filename
  * and removes the local server url prefixes.
  */
-export function createCapacitorRewriteFrames(): Integration {
-  const rewriteFrames = new RewriteFrames({
-    iteratee: (frame: StackFrame) => {
-      if (frame.filename) {
-        const isReachableHost = /^https?:\/\//.test(frame.filename);
-        const serverUrl = getCurrentServerUrl();
-        if (serverUrl) {
-          frame.filename = frame.filename.replace(serverUrl, '');
-        } else {
-          frame.filename = frame.filename
-            .replace(/^https?:\/\/localhost(:\d+)?/, '')
-            .replace(/^capacitor:\/\/localhost(:\d+)?/, '')
-        }
-        frame.filename = frame.filename.replace(/^ng:\/\//, '');
+export const capacitorRewriteFramesIntegration = (): Integration => {
+  return originalRewriteFramesIntegration({
+    iteratee: rewriteFramesIteratee,
+  });
+};
 
-        const isNativeFrame = frame.filename === '[native code]' || frame.filename === 'native';
+export const rewriteFramesIntegration = defineIntegration(
+  capacitorRewriteFramesIntegration,
+);
```

Comment:
Why do we use the `defineIntegration` here? The `capacitorRewriteFramesIntegration` already return generic `Integration` interface.